fractal_server/app/routes/api/v2/_aux_functions_tasks.py

```diff
@@ -226,11 +231,22 @@ async def _verify_non_duplication_user_constraint(
     res = await db.execute(stm)
     duplicate = res.scalars().all()
     if duplicate:
+        res = await db.execute(
+            select(CollectionStateV2).where(
+                CollectionStateV2.taskgroupv2_id == duplicate[0].id
+            )
+        )
+        state = res.scalars().one_or_none()
+        if (state is not None) and (state.data["status"] != "OK"):
```

Comment:
Use `data.get`

Reply:
Let's restructure this as follows

IF
state is not none 
and state.data has "status" key
and state.data["status"] is one of the "ongoing"

THEN
we can definitely say that one collection is ongoing

ELSE
we don't know, and we don't add any information to the error message